src/test/unit/service/filterService.test.ts

```diff
@@ -0,0 +1,64 @@
+import {FilterService} from '../../../main/service/filterService';
+import fs from 'fs';
+import path from 'path';
+
+const filterService = new FilterService();
+
+const rawData = fs.readFileSync(path.resolve(__dirname, '../mocks/courtAndHearings.json'), 'utf-8');
+const listData = JSON.parse(rawData);
+
+const crownCourt = 'Crown Court';
+const royalCourt = 'Royal Court';
+const jurisdiction = 'Jurisdiction';
+const filterOptions = {Jurisdiction: {'Crown Court': {checked: true}, Crown: {checked: false}}, Region: {Bedford: {checked: false}, Hull: {checked: true}}};
+
+describe('Filter Service', () => {
```

Comment:
No tests for getFilterValueOptions?

Reply:
no its a private method? you dont test private methods directly you test them passively